backend/lcfs/web/api/transfer/services.py

```diff
@@ -106,4 +112,20 @@ async def update_transfer(self, transfer_data: TransferUpdate) -> TransferSchema
                 transfer.comments = Comment(comment=transfer_data.comments)
 
         updated_transfer = await self.repo.update_transfer(transfer)
-        return TransferSchema.model_validate(updated_transfer)
\ No newline at end of file
+        return TransferSchema.model_validate(updated_transfer)
+
+    @service_handler
+    async def update_transfer(self, transfer_id: int, transfer_data: TransferUpdate) -> TransferSchema:
+        transfer = await self.repo.get_transfer_by_id(transfer_id)
+        if not transfer:
+            raise DataNotFoundException(
+                f"Transfer with ID {transfer_id} not found")
+        transfer.transfer_status_id = transfer_data.transfer_status_id
```

Comment:
this value has recently been updated to current_status_id so we'll need to swap that out here. Reason being is the TransferHistory table will keep the change history for each transfer, ie when their status changes. Please add the creation of a new TransferHistory record here with the new status, and set current_status on the Transfer record.